bin/node-pg-migrate.ts

```diff
@@ -22,17 +23,19 @@ process.on('uncaughtException', (err) => {
   process.exit(1);
 });
 
-const crossRequire = createRequire(resolve('_'));
-
 /**
- * Try to require a module and return null if it doesn't exist.
+ * Try to import a module and return null if it doesn't exist.
  *
- * @param moduleName The name of the module to require.
+ * @param moduleName The name of the module to import.
  */
-function tryRequire<TModule = unknown>(moduleName: string): TModule | null {
+async function tryImport<TModule = unknown>(
+  moduleName: string
+): Promise<TModule | null> {
   try {
-    return crossRequire(moduleName);
+    const module = await import(moduleName);
+    return module.default || module;
   } catch (error) {
+    console.log(`Error importing module ${moduleName}:`, error);
     if (
       // @ts-expect-error: TS doesn't know about code property
       error?.code !== 'MODULE_NOT_FOUND'
```

Comment:
`Error importing module config: Error [ERR_MODULE_NOT_FOUND]: Cannot find package 'config' imported from...`

```suggestion
      error?.code !== 'ERR_MODULE_NOT_FOUND'
```